.github/workflows/push-foundation-repo.yml

```diff
@@ -1,11 +1,16 @@
-name: Sync Cardinal-Cryptography repo with Aleph-Zero-Foundation repo
+name: Publish AlephBFT main to crates.io.
 
 on:
-  workflow_run:
-    workflows: ["Autobump version"]
-    branches: [main]
-    types:
-      - completed
+  # NOTE: changed to "manual workflow dispatch" for now.
+  # TODO: To be automated again, when workflows stabilize
+  # and pushes to main origin will be forbidden.
+  workflow_dispatch:
+
+#  workflow_run:
+#    workflows: ["Autobump version"]
+#    branches: [main]
+#    types:
+#      - completed
```

Comment:
Commented code.

Reply:
By purpose - see note above.

Reply:
It's not about history - commented code is left there, because it is the right code to be enabled in the future, when workflows stabilize.

Reply:
Yeah, but keeping commented code is always confusing. I would rather trust the person who will enable the automation here to do this correctly. If they need help one of the options is to look in the history, that's what I meant by it staying there.